package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-youtube",
-  "version": "0.9.7-noutube2",
+  "version": "0.9.7-noutube2-2",
```

Comment:
would have gone with `0.9.7-noutube3` but ok

Reply:
Haha I checked and this was the first version so thought it was referencing NoUTube 2, as in the non-PHP version